octobot_trading/exchanges/data/exchange_config_data.py

```diff
@@ -107,10 +104,10 @@ def _set_config_time_frame(self):
                 self.traded_time_frames.append(time_frame)
         if not self.exchange_manager.is_backtesting:
             # add shortest timeframe for realtime evaluators
-            client_shortest_time_frame = find_min_time_frame(self.exchange_manager.client_time_frames, MIN_EVAL_TIME_FRAME)
-            if client_shortest_time_frame not in self.traded_time_frames:
-                self.traded_time_frames.append(client_shortest_time_frame)
-
+            client_shortest_time_frame = find_min_time_frame(self.exchange_manager.client_time_frames,
+                                                             MIN_EVAL_TIME_FRAME)
+            self.real_time_time_frames.append(client_shortest_time_frame)
+        self.traded_time_frames.extend([tf for tf in self.real_time_time_frames if tf not in self.traded_time_frames])
```

Comment:
We should probably use `list(set(self.real_time_time_frames) & set(self.traded_time_frames))` instead.

Reply:
I also thought about it and I don't think creating a temporary set is faster or more clear that this, maybe you have more knowledge about it

Reply:
We should test because here you iterate on `self.traded_time_frames` for each `self.real_time_time_frames`, I think we can improve this.

Reply:
https://github.com/Drakkar-Software/OctoBot-Benchmarks/pull/4